src/ducks/components/App.jsx

```diff
@@ -14,8 +14,10 @@ export const App = () => (
     <main className='sto-content'>
       <Switch>
         <Route path='/discover' component={Discover} />
-        <Route path='/myapps' component={MyApplications} />
         <Route path='/install' component={HiddenInstallerView} />
+        <Route path='/myapps/:appSlug/:manage' component={Application} />
```

Comment:
Since `react-router` v4, we doesn't put all routes and nested routes in the same place like here. Nested routes should be defined in the child components like for Discover and MyApplications components.